File: 027_Remove_Element/027_1.py
from typing import List, Dict

class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        j=len(nums)
        for i in range(j-1,-1,-1):
            if nums[i] == val:
                nums.pop(i)    
        return len(nums)

File: 027_Remove_Element/027_1.c
int removeElement(int* nums, int numsSize, int val){
    if (numsSize == 0) return 0;
    int i = 0;
    int j = numsSize;
    while (i < j) {
        if (nums[i] == val) {
            nums[i] = nums[j - 1];
            j--;
        } else {
            i++;
        }
    }
    return i;
}